applications/launchpad/gui-react/src/store/mining/selectors.ts

```diff
@@ -21,3 +23,24 @@ export const selectLastSession = createSelector(
       ? miningState[node].sessions[miningState[node].sessions.length - 1]
       : undefined,
 )
+
+/**
+ * Is any mining node running?
+ * @returns {boolean}
+ */
+export const selectTariMiningStatus = (state: RootState) =>
+  state.mining.tari.status
+
+/**
+ * Is any mining able to run?
+ * (Is not in unknown, error or setup_required state)
+ */
+export const selectCanAnyMiningNodeRun = (state: RootState) =>
+  !!(
```

Comment:
this casting is not necessary, you're doing Boolean || Boolean (includes returns Boolean